trigger/payloads.go

```diff
@@ -438,7 +438,7 @@ func getFailedTestsFromTemplate(t task.Task) ([]task.TestResult, error) {
 }
 
 func taskLink(uiBase string, taskID string, execution int) string {
-	return fmt.Sprintf("%s/task/%s/%d/?redirect_spruce_users=true", uiBase, url.PathEscape(taskID), execution)
+	return fmt.Sprintf("%s/task/%s/%d?redirect_spruce_users=true", uiBase, url.PathEscape(taskID), execution)
```

Comment:
This doesn't seem to change the double forward slash at the beginning of the path. Can you double check on staging and validate that it is no longer the case. We had a similar case like this in the past and it is not a valid link. https://github.com/evergreen-ci/gimlet/pull/97#discussion_r713430345

Reply:
there was an extra / in the admin page that the prod page didn't have so i fixed it (updated the picure)

Reply:
the fix was changing base url on the admin page. if we want a validation we would make it there but annie said there's no need since we never change it